packages/menu/src/SubMenu.tsx

```diff
@@ -172,7 +172,7 @@ interface SubMenuProps {
   /**
    * Content to appear below main text of SubMenu
    */
-  description?: React.ReactElement;
+  description?: string | React.ReactElement;
```

Comment:
the story file wasn't getting type-checked because it was a .js file. type-checking revealed that this prop wasn't typed correctly.